src/main/scala/Player.scala

```diff
@@ -0,0 +1,6 @@
+case class Player(
+                   name: String,
```

Comment:
for indendation I would recommend to stick to 2 space everywhere, column formatting of code often ends with lots of lines needing reformatting for a simple change elsewhere e.g. rename the player class to something longer/shorter means you have to reindent all the lines.